client/src/app/globals.css

```
@@ -349,3 +349,53 @@ template {
 [hidden] {
   display: none;
 }
+
+/* Primitives
+   ========================================================================== */
+:root {
+  --default-font-family: "SUIT-Regular", sans-serif;
+  --default-font-style: normal;
+  --default-line-height: normal;
+  --spacing05: 4px;
```

```suggestion
  --primitives_spacing-05: 4px;
```

피그마에 primitives와 그렇지 않은 값이 나눠져있는 것 같은데, 우리도 구분하면 좋을 듯. 임의로 `_`로 primitives여부를 구분하고 `-`로 값을 구분하게 적어보았는데 어떰? 다른 방식도 괜찮고 (근데 css variables에 허용되는 특수문자가 별로 없긴 함ㅠ)
또 파일이 너무 길어진다면 globals_token.css 와 globals_primitives.css를 나누는 것도 괜찮을듯

---

아 피그마에 토큰 탭이 따로 있던 걸 못 봤군요 죄송...제안한 대로 token primitive 파일 나누어 컴포넌트 작업할 때는 token에만 의존하게 하면 좋을 거 같슴다! 수정할게용